spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/TradeRepository.java

```diff
@@ -35,6 +35,14 @@
 
 	int countByAction(String action);
```

Comment:
We need a test for something like this, as discussed during the review. Or is this just not something that can be easily supported? Is it supported in MongoDB or Cassandra?
```java
@Query("select plate from cars")
List<String> getPlates();
```

Reply:
This is definitely not being supported (and will need to be determined if at all, I have a question noted for the Spring Data team) in this PR. This PR is just about counts and exists queries.

Reply:
I bring this up because, theoretically, returning a non-collection should also be possible when the result set is of size 1.
For example.
```java
String findFirstNameByLastname(String lastname)
```
Do we support this?

Reply:
No, but I will make an issue for it: 

https://github.com/spring-cloud/spring-cloud-gcp/issues/1058

Reply:
@meltsufin PTAL. incidentally #1058 is also addressed. Now the SQL queries have no concept of count or exists, and it works with `String` or `List<String>` return types.